parsec/core/gui/tr/parsec_en.po

```diff
@@ -616,6 +616,9 @@ msgstr "The invitee has reset the process. We will restart from the beginning."
 msgid "TEXT_USER_PROFILE_STANDARD"
 msgstr "Standard"
 
+msgid "NOT_ALLOWED_OUTSIDER_PROFILE_TOOLTIP"
+msgstr "Disabled because backend does not allow it in its current configuration"
```

Comment:
This message seems too technical. I would rather go with something like:
"Your organization does not authorize this kind of profile." (closer to the French translation).